bin/alluxio

```diff
@@ -257,13 +257,8 @@ function main {
     runJavaClass "$@"
   ;;
   "runTest")
+    PARAMETER="configured" # instructs runTests to use the configured master address
```

Comment:
If we don't mind breaking backcompat here, we could completely drop the masterAddress option and always get it from configuration. It's currently awkward to use with fault-tolerance since you need to determine the leader address yourself and then bypass zookeeper. Would this work for you @jsimsa?